### PR TITLE
Python string format

### PR DESCRIPTION
**Pull request by Timple**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Ticket(s) this addresses   | https://github.com/ros-planning/navigation2/pull/2411#issuecomment-863561181 |
| Primary OS tested on | Ubuntu |
| Robotic platform tested on | None |

---

## Description of contribution in a few bullet points
As per @SteveMacenski's request: https://github.com/ros-planning/navigation2/pull/2411#issuecomment-863561181

---

#### For Maintainers: 
- [x] Check that any new parameters added are updated in navigation.ros.org
- [x] Check that any significant change is added to the migration guide
- [x] Check that any new functions have Doxygen added
- [x] Check that any new features have test coverage
- [x] Check that any new plugins is added to the plugins page
- [x] If BT Node, Additionally: add to BT's XML index of nodes for groot, BT package's readme table, and BT library lists

### CODE REVIEW

File: nav2_simple_commander/test/test_flake8.py [hunk 1]
```diff
@@ -21,5 +21,5 @@
 def test_flake8():
     rc, errors = main_with_errors(argv=[])
     assert rc == 0, \
-        'Found %d code style errors / warnings:\n' % len(errors) + \
```
Comment (SteveMacenski) [inline nav2_simple_commander/test/test_flake8.py:24]: The `test/` entries are autogenerated by colcon and aren't to be hand modified.

**Review by SteveMacenski — CHANGES_REQUESTED** [review]
Other than this one file, LGTM. Though I am not a python developer primarily, so I'd like a review from someone that does that more than I

File: nav2_bringup/bringup/launch/multi_tb3_simulation_launch.py [hunk 2]
```diff
@@ -123,7 +123,7 @@ def generate_launch_description():
     # Define commands for launching the navigation instances
     nav_instances_cmds = []
     for robot in robots:
-        params_file = LaunchConfiguration(robot['name'] + '_params_file')
+        params_file = LaunchConfiguration(f"{robot['name']}_params_file")
```
Comment (klintan) [inline nav2_bringup/bringup/launch/multi_tb3_simulation_launch.py:126]: I would probably try to keep single / double quotes consistent across the code base. If not for any other specific reason, this discussion could be valuable https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python (but my personal preference is just to make sure to be consistent, given there are no double / single quotes in the text itself)

**Review by klintan — COMMENTED** [review]
Looks good to me :) one minor comment.

File: nav2_bringup/bringup/launch/multi_tb3_simulation_launch.py (hunk 2 above)
Comment (SteveMacenski) [reply nav2_bringup/bringup/launch/multi_tb3_simulation_launch.py:126]: Oh yeah, @timple make sure you're running the linters and this passes in the python ones

Comment (Timple) [reply nav2_bringup/bringup/launch/multi_tb3_simulation_launch.py:126]: @klintan: Agreed on the consistency, changed.

@SteveMacenski: I doubt these kind of CI errors are related to the python strings?
```
/opt/overlay_ws/src/navigation2/nav2_dwb_controller/dwb_critics/test/obstacle_footprint_test.cpp:246
Expected equality of these values:
  critic->lineCost(3, 3, 0, 50)
    Which is: 85
  50
```

`ament_pep257` succeeds locally.

Comment (SteveMacenski) [reply nav2_bringup/bringup/launch/multi_tb3_simulation_launch.py:126]: No, other issues, have been fixed now but your fork is probably outdated. Merging! Thanks!

**Review by SteveMacenski — APPROVED** [review]

**Pull request merged**